recipes/python/python3.yaml

```diff
@@ -90,12 +90,15 @@ multiPackage:
             rsync -a "${BOB_DEP_PATHS[python::python3-wheel]}/" install/
 
         packageScript: |
-            autotoolsPackageTgt
+            autotoolsPackageBin
+            autotoolsPackageDev
```

Comment:
Hmm, now we always install the headers. Could there be separate -dev package instead? I have to admit I didn't think it through, though.

Reply:
now it looks like a common installation in a host system. installing the header will not have impact to the final image if we just install "*-tgt" in the end?

Reply:
At least on Debian there are different packages for the python interpreter, the cpython headers and pip/setuptools. I think we should provide this distinction too. But I'm not sure what this means in our context and how it fits.